src/lib/utils/jsonResponses/MultipartUploadStatus.tsx

```diff
@@ -0,0 +1,11 @@
+// https://docs.synapse.org/rest/org/sagebionetworks/repo/model/file/MultipartUploadStatus.html
+export type MultipartUploadStatus = {
+  uploadId: string,
+  startedBy: string,
+  startedOn: string,
+  updatedOn: string,
+  partsState: string,
+  state: 'UPLOADING' | 'COMPLETED' // The state of this file upload
+  resultFileHandleId: string,
+  clientSidePartsState?: boolean[]
```

Comment:
Could write a comment explaining that clientSidePartsState this is a utility to read over the partsState which is represented by the string with 1s and 0s